### PR TITLE
Applied rubocop to recently merged PRs

### PR DESCRIPTION
**Pull request by robert-w-gries**
Rubocop is now up-to-date with all the latest develop code.

### CODE REVIEW

**Comment by glennmatthews** [comment]
Looks like Travis is failing on Ruby 1.9.3 and 2.0.0. I think we should drop 1.9.3 since the node_utils gem no longer supports 1.9.3 - but can you look into why 2.0.0 is failing?

File: lib/puppet/provider/network_snmp/nxapi.rb [hunk 1]
```diff
@@ -26,34 +26,34 @@
...
-    @network_snmp = Cisco::SnmpServer.new()
+    @network_snmp = Cisco::SnmpServer.new
     @property_flush = {}
-    debug "Created provider instance of network_snmp"
+    debug 'Created provider instance of network_snmp'
   end
 
-  def self.get_properties
-    network_snmp = Cisco::SnmpServer.new()
+  def self.get_properties # rubocop:disable Style/AccessorMethodName
```
Comment (glennmatthews) [inline lib/puppet/provider/network_snmp/nxapi.rb]: Weren't we renaming these to 'properties_get'? Or am I misremembering?

Comment (robert-w-gries) [inline lib/puppet/provider/network_snmp/nxapi.rb]: Rubocop complains about accessor method names that start with get_ or set_, but this isn't an accessor method so I disabled the cop for this line

**Comment by glennmatthews** [comment]
Rob, I missed this in the previous PR - make sure you update docs/README-develop-types-providers so that the examples therein reflect the Rubocop changes.

File: lib/puppet/provider/ntp_config/nxapi.rb [hunk 2]
```diff
@@ -75,22 +75,25 @@ def self.prefetch(resources)
...
 
-      fail ArgumentError,
-        "The parameter 'source_interface' must not contain any uppercase characters" \
-          if @resource[:source_interface] =~ /[A-Z]+/
+    if @resource[:source_interface] =~ /\s+/
+      fail ArgumentError, "The parameter 'source_interface' must not contain" \
+                          'any spaces'
     end
+
+    return unless @resource[:source_interface] =~ /[A-Z]+/
```
Comment (robert-w-gries) [inline lib/puppet/provider/ntp_config/nxapi.rb]: It was hard to appease Rubocop in this method.  It doesn't like having multiple conditional blocks in a method and would rather you "return unless ...".

Are the end results of this change readable enough?

**Comment by robert-w-gries** [comment]
I don't think the travis error is related to my code.  I can successfully build on my puppet server.

The error message is:

    rake aborted!
    Can't find a puppet version.
    /home/travis/build/cisco/cisco-network-puppet-module/Rakefile:32:in `block in <top (required)>'

Seems to be an issue with puppet on the travis server.

File: lib/puppet/provider/ntp_config/nxapi.rb (hunk 2 above)
Comment (glennmatthews) [inline lib/puppet/provider/ntp_config/nxapi.rb]: Yeah, this works for me.

**Comment by robert-w-gries** [comment]
@glennmatthews 

> Rob, I missed this in the previous PR - make sure you update docs/README-develop-types-providers so that the examples therein reflect the Rubocop changes.

Done

File: lib/puppet/provider/network_snmp/nxapi.rb (hunk 1 above)
Comment (robert-w-gries) [inline lib/puppet/provider/network_snmp/nxapi.rb]: Right, we decided on property_get and property_set a while back.  Submitted new diff

**Comment by glennmatthews** [comment]
Sorry - if you look at feature/bgp, it's actually 'properties_get' not 'property_get' that's the convention.

**Comment by robert-w-gries** [comment]
Renamed again :) Changed property_set to properties_set too

**Pull request merged**